tests/test_reconcile_devin_wiki_access.py

```
@@ -0,0 +1,165 @@
+        )
+
+        installations = list_devin_installations(client)  # type: ignore[arg-type]
+
+        self.assertEqual([Installation(202, "timerloggedout-spec", "selected")], installations)
+        self.assertEqual([("GET", "user/installations?per_page=100")], client.calls)
+
+    def test_dry_run_reports_missing_selected_access_without_writing(self) -> None:
+        repository = Repository(42, "timerloggedout-spec/new-repository", "master", False)
+        installation = Installation(202, "timerloggedout-spec", "selected")
+        client = FakeClient()
+        with patch("reconcile_devin_wiki_access.list_accessible_repositories", return_value=[repository]), patch(
```

Pylint (4.0.6)

[convention] Line too long (113/100)

(C0301)